### ISSUE TITLE
[Bug]: Competitions: Event never changed the status

### ISSUE THREAD

**Issue by jonsnowpt** [Betarena/scores#1560]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### 🐞 Describe the bug

After the fixture was finished, the event never changed the status:

https://betarena-scores-platform.herokuapp.com/competitions/predictor/team-qaraba-will-win/42

![Screenshot 2023-09-01 at 08 49 43](https://github.com/Betarena/scores/assets/37311649/20563871-73ef-4649-9429-ef0f94030bf1)

### 🤔 Expected Behavior

The event should have changed the status, and the WIN icon should have appeared in the winner group.

### What environment is the bug/issue detected in.

- [ ] Production (scores.betarena.com)
- [X] Development

### What browsers are you seeing the problem on?

- [X] Firefox
- [ ] Chrome
- [ ] Safari
- [ ] Microsoft Edge
- [ ] Brave
- [ ] Opera

### What device/platform can the problem be seen?

- [X] 📱 Mobile
- [X] 💻 Tablet
- [X] 🖥️ Laptop/Desktop

### Steps To Reproduce

Visit the competition:

https://betarena-scores-platform.herokuapp.com/competitions/predictor/team-qaraba-will-win/42

Fixture:

https://scores.betarena.com/football/comerciantes-unidos-alianza-universidad-18856593

**Comment by migbash** [comment]
**🔸 UPDATE**

This took place because the `backend` for `livescores_now` did not pick up the change from `LIVE -> FT` for this fixture with `id: 18856593`. 

**LOGS ENDPOINT_5s_livescore_now_update.log :**

```bash
[...]
🟩 INFO: Triggered at: 08/31/2023 20:18:06 ➡ INFO: Fixture 18856593 status: NS -> LIVE 
[...]
🟩 INFO: Triggered at: 08/31/2023 21:05:03 ➡ INFO: Fixture 18856593 status: LIVE -> HT
[...]
🟩 INFO: Triggered at: 08/31/2023 21:22:00 ➡ INFO: Fixture 18856593 status: HT -> LIVE
[...]
❓ Missing LIVE -> FT
```

However, it appears it was correctly updated most likely later on, due to a delay from Sportmonks, with the correct status is shown on the `historic_fixtures`.

> **Note**
> Moving this issue to **🧊 Icebox** as required some looking into this issue and how to solve future similar cases, so all competitions are not lagging behind for **live** accurate data.

**Comment by migbash** [comment]
**🔸 UPDATE**

Happened again, got the same issue for a different competition / fixture, with the exact same issue, with the `LIVE -> FT` missing for fixture with `id: 18791254`.

**LOGS ENDPOINT_5s_livescore_now_update.log :**


```bash
[...]
🟩 INFO: Triggered at: 09/03/2023 21:31:04 ➡ INFO: Fixture 18856593 status: NS -> LIVE 
[...]
🟩 INFO: Triggered at: 09/03/2023 22:19:08 ➡ INFO: Fixture 18856593 status: LIVE -> HT
[...]
🟩 INFO: Triggered at: 09/03/2023 22:36:07 ➡ INFO: Fixture 18856593 status: HT -> LIVE
[...]
❓ Missing LIVE -> FT
```

> **Note**
> Promoting this issue back to **🟩 In Progress** , due to it being a repeating issue.

**Comment by jonsnowpt** [comment]
@migbash 

Next competition I'm checking the provider and Firebase status, since I'm finding it odd to get this status missing.

**Comment by migbash** [comment]
@jonsnowpt 

Any update on this ?

**Comment by jonsnowpt** [comment]
@migbash 

The last competition today morning was already removed from Livescore; I'm checking it with a new competition today.

**Comment by migbash** [comment]
@jonsnowpt 

Any updates on this since this `morning` ?

**Comment by jonsnowpt** [comment]
@migbash 

Since we didn't push anything for testing, we did not create other competition.

**Comment by migbash** [comment]
**🔸 UPDATE**

Happened again, got the same issue for a different competition / fixture, with the exact same issue, with the `LIVE -> FT` missing for fixture with `id: 18521206`.

**LOGS ENDPOINT_5s_livescore_now_update.log :**


```bash
[...]
🟩 INFO: Triggered at: 09/06/2023 19:06:01 ➡ INFO: Fixture 18521206 status: NS -> LIVE 
[...]
🟩 INFO: Triggered at: 09/06/2023 19:53:00 ➡ INFO: Fixture 18521206 status: LIVE -> HT
[...]
🟩 INFO: Triggered at: 09/06/2023 20:11:04 ➡ INFO: Fixture 18521206 status: HT -> LIVE
[...]
❓ Missing LIVE -> FT
```

> **Note**
> It has been verified and checked that both `livescores_now` have the correct fixture status of `FT`. However, it is not picked up by the `logs` and not appear. Also `Hasura` appears to be correct. A few issues were detected that could contribute to the issue occurring with the `status` update of the `competition`, outlined below:

```bash
[...]
09/06/2023 20:59:00 | Process: ENDPOINT_5s_livescore_now_update | OPTIONS:◈ env: prod
09/06/2023 20:59:00 | Process: ENDPOINT_livescores_now_scoreboard | OPTIONS:◈ env: prod
09/06/2023 21:00:06 | Process: ENDPOINT_5s_livescore_now_update | OPTIONS:◈ env: prod
09/06/2023 21:00:06 | Process: ENDPOINT_fixtures_check | OPTIONS:◈ env: prod
09/06/2023 21:00:06 | Process: END_F_DEV_HF_FL_A | OPTIONS:◈ env: prod
09/06/2023 21:00:06 | Process: ENDPOINT_livescores_now_scoreboard | OPTIONS:◈ env: prod
09/06/2023 21:00:06 | Process: END_F_HF_ML_A | OPTIONS:◈ env: prod
09/06/2023 21:00:06 | Process: ENDPOINT_scores_header_fixtures_information | OPTIONS:◈ env: prod
09/06/2023 21:00:06 | Process: ENDPOINT_historic_fixtures_2h_updated | OPTIONS:◈ env: prod
09/06/2023 21:00:06 | Process: END_F_P_L | OPTIONS:◈ env: prod
❌ Error: Value cannot be null. (Parameter 'input')
❌ Stacktrace:    at System.Text.RegularExpressions.ThrowHelper.ThrowArgumentNullException(ExceptionArgument arg)
   at backend_betarena_v2.Services.Data.Football.Player.GeneratePlayerUrls(Dictionary`2 data1, Int32 player_id, String player_name) in /src/Services/Data/Football/Data.Player.cs:line 353
   at backend_betarena_v2.Services.Data.Football.Player.PlayersUpsertV3Async(List`1 data1, IEnumerable`1 data2, String env) in /src/Services/Data/Football/Data.Player.cs:line 188
   at backend_betarena_v2.Services.Entry.Entry_B_SFP.Entry_B_SFP_M1_Async(Parameters parameters) in /src/Services/Entry/Entry.Players.cs:line 272
   at backend_betarena_v2.Services.Entry._TEMPLATE.MainEntry_01_Async(Func`2 fn, String methodName, String env, Parameters parameters) in /src/Services/Entry/_TEMPLATE.cs:line 85
📧 [ALERT] Email sent
❌ Error: Nullable object must have a value.
❌ Stacktrace:    at backend_betarena_v2.Services.Data.Competitions.CompetitionsData.Update_Competition_Status_Mod_Async(Dictionary`2 dataDict_1, List`1 dataList_1, Boolean enablePersistance)
   at backend_betarena_v2.Services.Entry.EntryCompetitions.EntryCompetitionsStatusUpdate_Async(Parameters parameters) in /src/Services/Entry/Entry.Competitions.cs:line 221
   at backend_betarena_v2.Services.Entry.Entry_B_G_LN.SubTask_01_Async(Parameters parameters, List`1 livescoreList, Dictionary`2 liveFixturesStatusDict) in /src/Services/Entry/Entry.Livescores.cs:line 261
   at backend_betarena_v2.Services.Entry.Entry_B_G_LN.Entry_B_G_LN_M_Async(Parameters parameters) in /src/Services/Entry/Entry.Livescores.cs:line 95
   at backend_betarena_v2.Services.Entry._TEMPLATE.MainEntry_01_Async(Func`2 fn, String methodName, String env, Parameters parameters) in /src/Services/Entry/_TEMPLATE.cs:line 85
📧 [ALERT] Email sent
[...]
``` 

With the `GeneratePlayerUrls` appearing quite frequently at times, but perhaps unrelated.

**Comment by migbash** [comment]
> **Note**
> Created offspring task ➤ [#233](https://github.com/Betarena/backend_betarena/issues/233) .

**Comment by migbash** [comment]
> **Note**
> Already addressed as part of issue ➤ [#233](https://github.com/Betarena/backend_betarena/issues/233) .
